src/status_im2/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.wallet.create-account.select-keypair.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.contexts.wallet.create-account.select-keypair.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def accounts
+  [{:account-props {:customization-color :turquoise
+                    :size                32
+                    :emoji               "\uD83C\uDFB2"
+                    :type                :default
+                    :name                "Trip to Vegas"
+                    :address             "0x0ah...71a"}
+    :networks      [{:name :ethereum :short-name "eth"}
```

Comment:
we have this data in the sub `wallet/network-details`

Reply:
oh actually we need to decide how to handle this as each account will have it's own network configuration. I'll open up a discussion off github. for now we can leave it hardcoded until we agree upon this approach 👍